packages/cli/src/tools/envinfo.ts

```diff
@@ -2,20 +2,37 @@
 import envinfo from 'envinfo';
 import {EnvironmentInfo} from '../commands/doctor/types';
 
-export default async function getEnvironmentInfo() {
-  return JSON.parse(
-    await envinfo.run(
-      {
-        Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
-        IDEs: ['Xcode', 'Android Studio'],
-        SDKs: ['iOS SDK', 'Android SDK'],
-        npmPackages: ['react', 'react-native', '@react-native-community/cli'],
-        npmGlobalPackages: ['*react-native*'],
-      },
-      {
-        json: true,
-        showNotFound: true,
-      },
-    ),
-  ) as EnvironmentInfo;
+/**
+ * Returns information about the running system.
+ * If `json === true`, or no options are passed,
+ * the return type will be an `EnvironmentInfo`.
+ * If set to `false`, it will be a `string`.
+ */
+async function getEnvironmentInfo(): Promise<EnvironmentInfo>;
+async function getEnvironmentInfo(json: true): Promise<EnvironmentInfo>;
+async function getEnvironmentInfo(json: false): Promise<string>;
+async function getEnvironmentInfo(
+  json = true,
+): Promise<string | EnvironmentInfo> {
+  const options = {json, showNotFound: json};
```

Comment:
I thought we always want to show Not Found?

Reply:
I'm all for always showing Not Found. Is just that in one place it was explicit and in the other not (and I think the default is `false`).

I'll update the PR

Reply:
Updated